src/Migrate.php

```diff
@@ -624,7 +625,17 @@ function (array $v) use ($database, $schemaName, $tableName) {
                     Helper::quoteIdentifier($view['schema_name'])
                 ));
 
-                $this->destinationConnection->query($view['text']);
+                try {
+                    $this->destinationConnection->query($view['text']);
+                } catch (Throwable $e) {
+                    $this->logger->warning(sprintf(
```

Comment:
We had a truncation problem dumping everything to stderr, (the event got truncated), so I switched the others to `info` instead.  Makes it harder to parse, but at least we catch them all